sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -103,6 +103,7 @@ private[this] object SharedFactory {
   val jsonFactory = new JsonFactoryBuilder()
```

Comment:
@HyukjinKwon Do you know why we don't use settings from JSONOptions:
https://github.com/apache/spark/blob/67b644c3d74b0587dd5d498b903383ac4de932fe/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala#L133-L147

Reply:
I think both functions were added mainly to support Hive compatibility, and I think that's why.

Reply:
I am not sure if we should match them or not for now considering the original intention of both functions.